src/actions.cpp

```diff
@@ -451,11 +331,37 @@ namespace Action {
         }
         std::cout << std::endl;
 
-        // Copyright
-        printTag(exifData, "Exif.Image.Copyright", _("Copyright"));
-
-        // Exif Comment
-        printTag(exifData, "Exif.Photo.UserComment", _("Exif comment"));
+        printTag(exifData, Exiv2::make                  , _("Camera make"));
+        printTag(exifData, Exiv2::model                 , _("Camera model"));
+        printTag(exifData, Exiv2::dateTimeOriginal      , _("Image timestamp"));
+        printTag(exifData, "Exif.Canon.FileNumber"      , _("File number"));
+        // From ExposureTime, failing that, try ShutterSpeedValue
```

Comment:
This code is tough to read/understand
```cpp
        // From ExposureTime, failing that, try ShutterSpeedValue
        printLabel(_("Exposure time"));
        bool done = 0 != printTag(exifData, Exiv2::exposureTime);   // do we mean done = printTag(....) != 0
        if (!done) {
            done = 0 != printTag(exifData, Exiv2::shutterSpeedValue); // # why are we setting done?
        }
        std::cout << std::endl;
        // Get if from FNumber and, failing that, try ApertureValue
        printLabel(_("Aperture"));
        done = 0 != printTag(exifData, Exiv2::fNumber);
        if (!done) {
            done = 0 != printTag(exifData, Exiv2::apertureValue); # why set done?
        }
        std::cout << std::endl;
```

Can we "hide" the details in another printTag() method which takes two easyaccess function selectors?  Then 14 lines of if statements can be expressed in two lines that match the pattern of code like this:

```cpp
printTag(exifData, Exiv2::exposureTime, Exiv2::shutterSpeedValue, _("Exposure time"));
printTag(exifData, Exiv2::fNumber     , Exiv2::apertureValue    , _("Aperture"));
```

When you add `printTag(exifData, Exiv2::fNumber     , Exiv2::apertureValue    , _("Aperture"));`, can you add spaces to the printTag's above and below, so that the whole thing is instantly readable.  Something like:

```cpp
        printTag(exifData, Exiv2::dateTimeOriginal                                , _("Image timestamp"));
        printTag(exifData, "Exif.Canon.FileNumber"                                , _("File number"));
        printTag(exifData, Exiv2::exposureTime          , Exiv2::shutterSpeedValue, _("Exposure time"));
        printTag(exifData, Exiv2::fNumber               , Exiv2::apertureValue    , _("Aperture"));
        printTag(exifData, Exiv2::exposureBiasValue                               , _("Exposure bias"));
        printTag(exifData, Exiv2::flash                                           , _("Flash"));
```

Reply:
Ok, I'll clean this up as soon as I get a chance. Maybe with a slight variation to avoid addition of yet another printTag() overload, so how about this single prototype instead?

``` cpp
int printTag(const Exiv2::ExifData& exifData,
                     EasyAccessFct easyAccessFct,
                     const std::string& label = "",
                     EasyAccessFct easyAccessFctFallback = NULL) const;
```

Then we can have e.g.:

``` cpp
        printTag(exifData, Exiv2::fNumber,            _("Aperture"),     Exiv2::apertureValue);
        printTag(exifData, Exiv2::exposureBiasValue,  _("Exposure bias") );
```

Otherwise I can put the fallback function argument before the label, but then I have to explicitly specify a lot of `NULL`s everywhere else... It's a style/readability preference:

``` cpp
        printTag(exifData, Exiv2::fNumber,           Exiv2::apertureValue, _("Aperture"));
        printTag(exifData, Exiv2::exposureBiasValue, NULL,                 _("Exposure bias"));
```

Flip a coin ;)

Reply:
Sure, good suggestions.  I like the optional fourth "fallback selector" parameter.

Now you see the benefit of having Team Exiv2 write access to our repos.  I could updated your PR and ask you to approve it.  Simpler work-flow.